ckanext/followy/tests/e2e/followy_test.py

```diff
@@ -68,16 +71,7 @@ def test_create_dataset_and_follow(self):
 
 
     def test_following_dataset(self):
-        self.browser.get('http://localhost:5000')
-        login = self.browser.find_element_by_link_text("Log in")
-        login.click()
-        username = self.browser.find_element_by_id("field-login")
-        password = self.browser.find_element_by_id("field-password")
-        login_Btn = self.browser.find_elements_by_class_name("btn-primary")[0]
-        username.send_keys("ghost")
-        password.send_keys("ghostmode")
-        login_Btn.click()
-        assert 'http://localhost:5000/dashboard' in self.browser.current_url
+        self.user_login()        
```

Comment:
This is a great approach for readability, the code looks cleaner now

Reply:
Ok, Thank you!